docs/UserGuide.md

```diff
@@ -58,6 +63,7 @@ Command | Parameters | Description
 `delete -OBJECT` | `OBJECT` activity/ accommodation/ person/ travelplan | Deletes the given object type
 `edit -OBJECT` | `OBJECT` activity/ accommodation/ person/ travelplan | Edits the details of the given object type
 `goto PLAN_NAME` | `PLAN_NAME` nameOfTravelPlan/ wishlist | Navigate to the specific travel plan/ wishlist
+`find -OBJECT` | `OBJECT` activity/ accommodation/ person | Finds the given object type whose names contain any of the given keywords
```

Comment:
i think for all the find  command in the table, we should change to `COMMAND -OBJECT KEYWORD`, then the parameters add in `KEYWORD find object that contains this keyword`

Reply:
I did this for my delete command, think we shld do it for all commands on the table. Add the additional parameter for the commands that require it. Like `delete -object index`